### PR TITLE
Add initial kubernetes services and configmap UI support

### PR DESCRIPTION
**Pull request by mclarke47**
Added UI components for showing retrieved services and configmaps

Screen Shot 2020-09-28 at 11 00 06
Screen Shot 2020-09-28 at 10 59 58

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [x] Prettier run on changed files
- [x] Tests added for new functionality
- [x] Regression tests added for bug fixes

### CODE REVIEW

**Comment by mclarke47** [comment]
Had a bit of a find and replace whoopsie 😂

File: plugins/kubernetes-backend/src/service/getKubernetesObjectsByServiceIdHandler.ts [hunk 1]
```diff
@@ -34,7 +34,6 @@ const DEFAULT_OBJECTS = new Set<KubernetesObjectTypes>([
   'pods',
   'services',
   'configmaps',
-  'secrets',
```
Comment (mclarke47) [inline plugins/kubernetes-backend/src/service/getKubernetesObjectsByServiceIdHandler.ts:37]: In order to not give people Authorization nightmares before we have that system in place. I think we should de-scope secrets until we have a separate strategy for viewing them safely.

Comment (Rugvip) [reply plugins/kubernetes-backend/src/service/getKubernetesObjectsByServiceIdHandler.ts:37]: Yep, sounds good

**Review by Rugvip — APPROVED** [review]
🎉

File: plugins/kubernetes/src/components/KubernetesContent/KubernetesContent.tsx [hunk 2]
```diff
@@ -17,23 +17,33 @@
...
+import { Services } from '../Services';
+import { ConfigMaps } from '../ConfigMaps';
 
-const findDeployments = (fetchResponse: FetchResponse[]): DeploymentTriple => {
+interface GroupedResponses extends DeploymentTriple {
+  services: V1Service[];
+  configMaps: V1ConfigMap[];
+}
+
+const groupResponses = (fetchResponse: FetchResponse[]) => {
```
[the remark this answers is not in the document]
Comment (mclarke47) [reply plugins/kubernetes/src/components/KubernetesContent/KubernetesContent.tsx:46]: Definitely gonna do that in the next PR. Don't like this function 😂

**Pull request merged**